docs/src/wallet-guide/apps.md

```diff
@@ -7,6 +7,30 @@ experience for most people who are new or experienced with using crypto wallets.
 currently the easiest and fastest way to get set up with a new wallet on Solana.
 The app is free and getting your wallet set up only takes a few minutes.
 
+### Trust Wallet Security
+
+Tokens held in Trust Wallet are only as secure as the device on which the app is
+installed.  Anyone who is able to unlock your phone or tablet may be able to
+use the Trust Wallet app and transfer your tokens.  It is *highly* recommended
```

Comment:
We should avoid making recommendations. Instead, how about something along the lines of,  "To improve security, you can add a passcode to the Trust Wallet application. To add a Trust Wallet passcode, open the app and go to Settings -> Security -> Passcode."